a10_neutron_lbaas/tests/unit/v2/test_handler_member.py

```diff
@@ -119,3 +120,12 @@ def test_delete_count_gt_one(self):
 
         self.a.last_client.slb.service_group.member.delete.assert_called_with(
             m.pool_id, name, m.protocol_port)
+
+    def test_updating_oper_stats(self):
+        status_check.status_update_v2(self.a)
+        self.a.last_client.slb.service_group.member.get_oper.mock_calls[0].assert_called_with(
```

Comment:
This is redundant given the test above.  I try to stick to 1 assertion per test - exceptions to this are: assert return value of call is not None, then assert what I'm actually testing.  This makes the test fail due to the value not being returned and provides clearer failure output.